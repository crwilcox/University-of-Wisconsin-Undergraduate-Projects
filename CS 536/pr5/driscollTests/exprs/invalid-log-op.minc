void main(int x)
{
  x || false;
}
