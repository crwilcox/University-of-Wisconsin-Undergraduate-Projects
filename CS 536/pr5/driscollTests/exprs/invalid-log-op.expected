3:5:Logical operator applied to non-bool operand
